scss/_pagination.scss

```diff
@@ -6,9 +6,21 @@
 //
 .breadcrumb {
   background-color: $breadcrumb-bg;
+
```

Comment:
Should this be optional?  Don't ask me.

Reply:
Same question about this class being in pagination.scss vs breadcrumb.scss

Reply:
Sorry if these are annoying questions ha

Reply:
`breadcrumb.scss` is under Node Modules. And according to Martin: "Nothing that's under node_modules should get edited: that's the directory tree where npm puts all the dependencies, including the unmodified upstream Bootstrap. So they don't show up in the https://github.com/az-digital/arizona-bootstrap view of the repository at all (the .gitignore file explicitly block changes to node_modules)."